client/components/publish-confirmation.vue

```diff
@@ -0,0 +1,69 @@
+<template>
```

Comment:
Probably the whole template should render something like 404 or 403 if user is not logged in (and in the future if no permissions).

Reply:
We could also add a route interceptor and redirect to the `NotFound` component or a new one for unauthorized access.

Reply:
See the user-signing component. I have this there.

Reply:
I would not for now do it at a route level, just we render access denied if user is not logged in. The opposite of the user-signing component.

Reply:
Got it!